Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3766,8 +3766,26 @@ script:
       name: device-group
     - description: Return raw XML
       name: is_xml
+    - name: is_file
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Return response as file.
+      defaultValue: "false"
+    - name: file_name
+      description: Optional file name when running this command with is_file=true.
+      defaultValue: pan-os-result-file
+    - name: polling
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Set to true when polling the status of a job command to see if it is complete.
```

Comment:
this description is misleading a bit, because `pan-os` command can be used to retrieve any kind of data and not only jobs statuses

Reply:
Right, but the `polling` arg that this description is for only applies when polling a job status.